pystac/stac_object.py

```diff
@@ -536,6 +538,14 @@ def to_dict(
         """
         raise NotImplementedError
 
+    def _repr_html_(self) -> str:
+        jinja_env = get_jinja_env()
+        if jinja_env:
+            template = jinja_env.get_template("JSON.jinja2")
+            return str(template.render(dict=self.to_dict()))
```

Comment:
```suggestion
            return str(template.render(dict=self.to_dict(transform_hrefs=False)))
```

This will prevent resolving the whole tree, which I'm pretty sure is what we want.

Reply:
ah yes!